### ISSUE TITLE
Work out colours

### ISSUE THREAD

**Issue by erkannt** [hivereview/thehive#273]
- [x] choose primary colour `#00796b` 
- [ ] choose logo (depends on finalising the name)
- [ ] ensure accessibility

**Comment by davidcmoulton** [comment]
Accessibility report for `#00796b`, showing the only contrast failure is at font sizes smaller than 18px for AAA level, which is fine as we're shooting for AA.
<img width="1186" alt="" src="https://user-images.githubusercontent.com/2893480/94941487-d9de5c00-04cc-11eb-8db2-c27a5221e611.png">

Generated by https://contrastchecker.com/.

**Comment by davidcmoulton** [comment]
Colours, branding, logo waiting on a name.
